tests/plan/lint/test.sh

```diff
@@ -52,6 +52,11 @@ rlJournalStart
         rlAssertGrep "fail repo 'http://invalid-url' cannot be cloned" \
             $rlRun_LOG
         rlRun "rm $rlRun_LOG"
+
+        rlRun -s "tmt plan lint invalid_attr" 1
+        rlAssertGrep "fail unknown attribute .* is used" $rlRun_LOG
+        rlRun "rm $rlRun_LOG"
```

Comment:
Is this test enough? Or should I add something more?

Reply:
Could you please also add a typo in a step name to the test? This way it only tests the core attributes but the typo can be e.g. `discove` or something along those lines

Reply:
Changes are already in the new commit, you can check it out.